theano/gof/tests/test_vm.py

```diff
@@ -331,6 +331,9 @@ def time_linker(name, linker):
 
 
 class RunOnce(theano.Op):
+
+    __props__ = ()
+    
```

Comment:
Is "return self.name" 
the same as "return self.__class__.__name__" (in test_link.py) ?